docs/CONTRIBUTING.md

```diff
@@ -66,6 +83,22 @@ After you are able to successfully build JsDbg, please follow the steps below to
 
 8. The JsDbg server will launch and prompt you to select a browser. If the debugger is attached to one browser, it is recommended that you choose a different browser for JsDbg.
 
+#### GDB
+
+1. Make sure you added JsDbg to your `.gdbinit` as described in the Linux build instructions above
```

Comment:
you "have" added

also missing a '.' at the end

Reply:
Done